apps/src/gamelab/GameLab.js

```diff
@@ -432,23 +447,6 @@ GameLab.prototype.onTick = function () {
 
 GameLab.prototype.handleExecutionError = function (err, lineNumber) {
 /*
-  if (!lineNumber && err instanceof SyntaxError) {
-    // syntax errors came before execution (during parsing), so we need
-    // to determine the proper line number by looking at the exception
-    lineNumber = err.loc.line;
-    // Now select this location in the editor, since we know we didn't hit
-    // this while executing (in which case, it would already have been selected)
-
-    codegen.selectEditorRowColError(studioApp.editor, lineNumber - 1, err.loc.column);
-  }
-  if (Studio.JSInterpreter) {
-    // Select code that just executed:
-    Studio.JSInterpreter.selectCurrentCode("ace_error");
-    // Grab line number if we don't have one already:
-    if (!lineNumber) {
-      lineNumber = 1 + Studio.JSInterpreter.getNearestUserCodeLine();
-    }
-  }
```

Comment:
I know it's commented, but I removed this code in App Lab and Studio, so it seemed appropriate to remove it here too.